searchcore/src/vespa/searchcore/proton/matching/match_master.cpp

```diff
@@ -30,18 +30,19 @@ namespace {
 struct TimedMatchLoopCommunicator : IMatchLoopCommunicator {
     IMatchLoopCommunicator &communicator;
     fastos::StopWatch rerank_time;
+    fastos::TimeStamp elapsed;
```

Comment:
stopwatch might not be the best fit here, consider explicit time sampling instead

Reply:
I will leave that for another pass on stopwatch.